test/FunctionalTests/Tests/DataEdmLib/FunctionalTests/CsdlParsingTests.cs

```diff
@@ -2376,14 +2376,49 @@ public void TestEnumTypeParsingOutOfRange()
 
             bool valid = model.Validate(out errors);
             Assert.IsFalse(valid, "valid");
-            Assert.AreEqual(1, errors.Count(), "one validation error expected");
+            Assert.AreEqual(2, errors.Count(), "two validation error expected");
             Assert.AreEqual(EdmErrorCode.EnumMemberValueOutOfRange, errors.ElementAt(0).ErrorCode, "EnumMemberValueOutOfRange expected");
+            Assert.AreEqual(EdmErrorCode.EnumMemberTypeMustMatchEnumUnderlyingType, errors.ElementAt(1).ErrorCode, "EnumMemberTypeMustMatchEnumUnderlyingType expected");
             Assert.IsTrue(errors.ElementAt(0).ErrorLocation.ToString().Contains("(4, 8)"), "error location");
+            Assert.IsTrue(errors.ElementAt(1).ErrorLocation.ToString().Contains("(4, 8)"), "error location");
 
-            IEdmEnumMember badEnumMemeber = ((IEdmEnumType)model.FindType("foo.Color")).Members.First(m => m.Name == "Green");
-            Assert.AreEqual(model.FindType("foo.Color"), badEnumMemeber.DeclaringType, "Bad member has correct type.");
-            Assert.IsTrue(badEnumMemeber.Value.IsBad(), "Bad member value is bad.");
-            Assert.AreEqual(EdmPrimitiveTypeKind.Int64, badEnumMemeber.Value.Type.PrimitiveKind(), "Bad members bad value has correct type.");
+            IEdmEnumMember enumMemeber = ((IEdmEnumType)model.FindType("foo.Color")).Members.First(m => m.Name == "Green");
+            Assert.AreEqual(model.FindType("foo.Color"), enumMemeber.DeclaringType, "Enum member has correct type.");
+            Assert.IsTrue(enumMemeber.Value.IsBad(), "Enum member value is bad.");
+            Assert.AreEqual(EdmPrimitiveTypeKind.Int64, enumMemeber.Value.Type.PrimitiveKind(), "Enum member bad value has correct type.");
+        }
+
+        [TestMethod]

```

Comment:
Not for your change.  But if we will create a new test class, consider using xUnit.